### PR TITLE
KACPER-1 port and db details stored in .env

### PR DESCRIPTION
**Pull request by kmkcc2**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.env
```
Comment (FilipAlbrycht) [inline .gitignore:2]: add .env.example with schema of .env (without values)

File: config/config.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "development": {
+      "username": process.env.DB_USERNAME,
+      "password": process.env.DB_PASSWORD,
+      "database": process.env.DB_DATABASE,
+      "host": process.env.DB_HOST,
+      "dialect": process.env.DB_DIALECT
+    },
+    "test": {
+      "username": "root",
+      "password": null,
```
Comment (FilipAlbrycht) [inline config/config.js]: use process.env for every single environment

File: models/index.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ const Sequelize = require('sequelize');
 const process = require('process');
 const basename = path.basename(__filename);
 const env = process.env.NODE_ENV || 'development';
-const config = require(__dirname + '/../config/config.json')[env];
+const config = require(__dirname + '/../config/config.js')[env];
```
Comment (FilipAlbrycht) [inline models/index.js]: const config = require('../config/config.js')[env]; - shorter version

**Review by FilipAlbrycht — CHANGES_REQUESTED** [review]

**Review by FilipAlbrycht — APPROVED** [review]

**Review by benmotyka — COMMENTED** [review]
ok

**Pull request merged**